src/api/types.ts

```diff
@@ -20,6 +25,7 @@ export interface IngestResponse {
 export interface Reference {
   source_filename: string;
   filename_md5: string;
+  status?: IngestStatus & string;
```

Comment:
Do you know why we have the extra `& string` in this definition?
Can we make this field mandatory? (I know we might need to reset the references file for this to work fine, but I think that it worth the trouble). Also, I saw above that the field has a default PROCESSING value.

Reply:
Great. Thanks.